mmseg/datasets/kitti_seg.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+# Register MyDataset class into DATASETS
+@DATASETS.register_module()
+class KittiSegDataset(CustomDataset):
+    CLASSES = ('road', 'sidewalk', 'building', 'wall', 'fence', 'pole',
+               'traffic light', 'traffic sign', 'vegetation', 'terrain', 'sky',
+               'person', 'rider', 'car', 'truck', 'bus', 'train', 'motorcycle',
+               'bicycle')
+
+    PALETTE = [[128, 64, 128], [244, 35, 232], [70, 70, 70], [102, 102, 156],
+               [190, 153, 153], [153, 153, 153], [250, 170, 30], [220, 220, 0],
+               [107, 142, 35], [152, 251, 152], [70, 130, 180], [220, 20, 60],
+               [255, 0, 0], [0, 0, 142], [0, 0, 70], [0, 60, 100],
+               [0, 80, 100], [0, 0, 230], [119, 11, 32]]
```

Comment:
Since the KITTI dataset is similar to Cityscapes, you may inherit from `CityscapesDataset` class directly.
For example: 
https://github.com/open-mmlab/mmsegmentation/blob/46326f63ce411c794d237e986dd3924590d0e75e/mmseg/datasets/dark_zurich.py#L7 and https://github.com/open-mmlab/mmsegmentation/blob/46326f63ce411c794d237e986dd3924590d0e75e/mmseg/datasets/night_driving.py#L7